### DISCUSSION TITLE
Headless option

### DISCUSSION THREAD

**Discussion by TimGeyssens** [TimGeyssens/Umbraco-v8-Community-Website-StarterKit#9] [General]
Would be ace if we also have a headless bit... probably using https://github.com/mattbrailsford/umbraco-headrest

So full site can just use the standard razor views but an additional small app can be done headless..

**Comment by CarlSargunar** [comment]
Always wanted to build a xamarin app which spoke to a headless CMS, so this could be a great option for that - only as an example exercise but yeah, for sure

**Reply by TimGeyssens** [reply]
yes, cool idea!

**Comment by louisjrdev** [comment]
It would be great to see a really good practice example for this, I would like to see something with a TS core that can be re-used in a [NextJS](https://nextjs.org)(React) or a [Nuxt](https://nuxtjs.org)(Vue) with SSR.

**Reply by TimGeyssens** [reply]
yes!
